File: UILabelDemo/UILabelDemo/ViewController.m
//
//  ViewController.m
//  UILabelDemo
//
//  Created by djstava on 11/10/2016.
//  Copyright © 2016 djstava. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    UILabel *lable = [[UILabel alloc] init];
    lable.text = @"Hello iOS!";
    lable.frame = CGRectMake(100, 100, 100, 40);
    
    lable.backgroundColor = [UIColor grayColor];
    
    lable.shadowColor = [UIColor redColor];
    lable.shadowOffset = CGSizeMake(5, 5);
    
    lable.font = [UIFont systemFontOfSize:20];
    lable.textColor = [UIColor yellowColor];
    lable.textAlignment = NSTextAlignmentCenter;
    lable.numberOfLines = 0;
    
    [self.view addSubview:lable];
    
    
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: TableViewDemo/TableViewDemo/ViewController.swift
//
//  ViewController.swift
//  TableViewDemo
//
//  Created by djstava on 2/17/16.
//  Copyright © 2016 djstava. All rights reserved.
//

import UIKit

class ViewController: UIViewController, UITableViewDataSource,UITableViewDelegate {
    
    var colors = ["Red","Yellow","Green","Gray","Orange","Black","White"]
    
    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        return colors.count
    }
    
    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
        let cell = tableView.dequeueReusableCellWithIdentifier("Cell", forIndexPath: indexPath)
        cell.textLabel?.text = colors[indexPath.row]
        return cell
    }

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

